homeassistant/components/unifi/controller.py

```diff
@@ -155,6 +156,15 @@ def event_update(self):
                 self.unifi_config = unifi_config
                 break
 
+        if (
+            CONF_DONT_TRACK_NEW_CLIENTS in self.unifi_config
+            and self.config_entry.system_options.disable_new_entities
+            != self.unifi_config[CONF_DONT_TRACK_NEW_CLIENTS]
+        ):
+            self.config_entry.system_options.disable_new_entities = self.unifi_config[
```

Comment:
We should call the `update` method to update it.

Reply:
btw reading the `update` method, it is not triggering a save of the config entry right now, it should.

Reply:
I don't know what I was thinking >.<